src/status_im/ui/screens/profile/visibility_status/styles.cljs

```diff
@@ -15,23 +16,22 @@
    :padding-right          12})
 
 (defn visibility-status-dot [dot-color size]
-  {:background-color dot-color
-   :width            size
-   :height           size
-   :border-radius    (/ size 2)
-   :border-width     3.5
-   :border-color     (quo2.colors/theme-colors quo2.colors/white quo2.colors/neutral-90)})
+  (if (<sub [:ui/old?])
```

Comment:
it's better to pass subs value from outside this function as a parameter

Reply:
Could you give me a bit more context as to why you think it's better to pass it as a parameter? Is it because  `**/styles.cljs` namespaces should not use re-frame subscriptions in general? Maybe it's an architectural decision (?), which would make sense if you ask me.

Reply:
yes subscriptions should be used only directly in the view, and not hidden in the styles or other functions, best way is to subscribe in `let` form in the view function

Reply:
https://github.com/status-im/status-mobile/pull/13747/commits/ed07d93804a7eabfba54fb4ed7f56699da2845fe